client/manage.html

```diff
@@ -61,11 +61,11 @@ <h2 class="text-center">Enter session details</h2>
               <span id="session-fail" class="fail-icon glyphicon glyphicon-remove form-control-feedback hidden"
                     aria-hidden="true"></span>
               <span id="session-fail-help"
-                    class="fail-help help-block hidden">Please input the 26-character session key.</span>
+                    class="fail-help help-block hidden">Please input the 26-character {{ client.session_key }}.</span>
```

Comment:
The custom session key does not actually show up in the UI. I don't think client is defined here.